tlslite/keyexchange.py

```diff
@@ -274,6 +275,9 @@ def processClientKeyExchange(self, clientKeyExchange):
             raise TLSLocalAlert(AlertDescription.illegal_parameter,
                                 "Invalid dh_Yc value")
 
+        if dh_Yc % self.dh_p == 0:
+            raise TLSIllegalParameterException("Suspicious dh_Yc value")
```

Comment:
> Because if I want to fulfill condition, I need dh_Yc >= dh_p

or `dh_Yc` == 0

but very good point, which make that this test completely redundant if we have the test in line 273...

so that code is not needed and can be removed, I don't think you need to add test case for it :)